test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -306,6 +306,11 @@ public static double randomDoubleBetween(double start, double end, boolean lower
         return result;
     }
 
+    /** A random float between 0.0 and 1.0 (inclusive). */
+    public static float randomFloatRatio() {
```

Comment:
@abeyad randomFloat() should do just that. The method name suggests something more generic but if you check it delegates to Random.nextFloat() which has some docs. I also just re-checked, just so make sure.